ansible/playbooks/nagios/roles/Nagios_Server/scripts/nagios_check.sh

```diff
@@ -0,0 +1,145 @@
+checks["Updates Required - apt"]="define service{
+        use                             generic-service
+        host_name                       $host_name
+	check_period			once-a-day-at-8
+        service_description             Updates Required - apt
+        check_command			check_by_ssh!/usr/lib/nagios/plugins/check_apt
+	notifications_enabled   0
+        }"
+checks["Updates Required - Zypper"]="define service{
+        use                             generic-service
+        host_name                       $host_name
+	check_period			once-a-day-at-8
+        service_description             Updates Required - Zypper
+	check_command                   check_by_ssh!/usr/local/nagios/libexec/check_zypper
+	notifications_enabled   0        
+	}"
+checks["Total Processes"]="define service{
+        use                             generic-service        
+        host_name                       $host_name
+        service_description             Total Processes
+        check_command                   check_by_ssh!/usr/lib/nagios/plugins/check_procs -w 185 -c 210
+        action_url      /nagiosgraph/cgi-bin/show.cgi?host=$HOSTNAME$&service=$SERVICEDESC$' onMouseOver='showGraphPopup(this)' onMouseOut='hideGraphPopup()' rel='/nagiosgraph/cgi-bin/showgraph.cgi?host=$HOSTNAME$&service=$SERVICEDESC$&period=hour&rrdopts=-w+450
+	}"
+checks["Zombie Processes"]="define service{
+        use                             generic-service        
+        host_name                       $host_name
+        service_description             Zombie Processes
+        check_command                   check_by_ssh!/usr/lib/nagios/plugins/check_procs -w 5 -c 10 -s Z
+        action_url      /nagiosgraph/cgi-bin/show.cgi?host=$HOSTNAME$&service=$SERVICEDESC$' onMouseOver='showGraphPopup(this)' onMouseOut='hideGraphPopup()' rel='/nagiosgraph/cgi-bin/showgraph.cgi?host=$HOSTNAME$&service=$SERVICEDESC$&period=hour&rrdopts=-w+450
+	}"
+checks["Current Users"]="define service{
+        use                             generic-service        
+        host_name                       $host_name
+        service_description             Current Users
+        check_command                   check_by_ssh!/usr/lib/nagios/plugins/check_users -w 5 -c 10
+        action_url      /nagiosgraph/cgi-bin/show.cgi?host=$HOSTNAME$&service=$SERVICEDESC$' onMouseOver='showGraphPopup(this)' onMouseOut='hideGraphPopup()' rel='/nagiosgraph/cgi-bin/showgraph.cgi?host=$HOSTNAME$&service=$SERVICEDESC$&period=hour&rrdopts=-w+450
+	}"
+checks["SSH"]="define service{
+        use                             local-service
+	host_name                       $host_name
+	service_description             SSH
+	check_command			check_ssh
+        action_url      /nagiosgraph/cgi-bin/show.cgi?host=$HOSTNAME$&service=$SERVICEDESC$' onMouseOver='showGraphPopup(this)' onMouseOut='hideGraphPopup()' rel='/nagiosgraph/cgi-bin/showgraph.cgi?host=$HOSTNAME$&service=$SERVICEDESC$&period=hour&rrdopts=-w+450
+	}"
+
```

Comment:
I think this needs rethinking, and the code made more dynamic, ideally it should parse the config file for a machine, and remove the check from the machines individual config file, this may be different for each machine, so managing a defined check list like this would be quite a overhead.

Reply:
Alright. I'll work on that.